themes/digital.gov/src/scss/new/_resources_featured.scss

```diff
@@ -61,12 +72,12 @@
       }
     }
   }
-  .collection{
-    @include u-margin-bottom(0);
+  .resources-content {
+    @include u-bg('white');
     @include u-padding-x(2);
     @include u-padding-y(3);
     @include at-media('tablet-lg') {
-      @include u-padding(0);
+      @include u-radius-top("lg");
     }
   }
 }
```

Comment:
Can we run prettier to fix spacing and quotes issues (inconsistent quotes).

Reply:
I hesitated to run somethign like Prettier since I thought it would change so many things, but I can give it a shot

Reply:
Understandable, if we can use consistent quotes that should be enough.

Reply:
But I did update to use single quotes for tokens!